src/instructlab/configuration.py

```diff
@@ -632,10 +636,23 @@ def init(
             config_obj.general.log_level = "DEBUG"
             config_obj.general.debug_level = debug_level
 
+        # --log-format option overrides config file
+        # If the flag is not used
+        # Only do this, if not called from "config" or "init" (support the legacy `ilab init` command)
+        if ctx.invoked_subcommand not in ("config", "init"):
```

Comment:
how useful is it to pass a custom logger via cli? can't we do the same by modifying a config file - or passing a custom one, if really needed?

Reply:
resolved with https://github.com/instructlab/instructlab/pull/1741#issuecomment-2252354702